spec/place_order_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'place_order'
+
+describe PlaceOrder do
+
+  describe "#takes_dishes" do
+    it { is_expected.to respond_to(:ask_for_order) }
+    it 'asks for an order' do
+      expect(subject.ask_for_order).to eq "Please place your order in the form of order number, quantity, and start on a new line for each item E.g. 1, 7[return key]2, 4,[return key][return key]"
+    end
+    it { is_expected.to respond_to(:take_order) }
+    it 'gets an order' do
+      # TODO - find a way to check actual method here
+      allow($stdin).to receive(:gets).and_return('foo')
+      test_order = $stdin.gets
+      expect(test_order).to eq('foo')
+    end
+    # edge cases to be dealt with - no input, impossible input, non integers, too many numbers, etc
+  end
+
+  describe "#calculates expected total" do
+    it { is_expected.to respond_to(:take_user_total) } 
+      #TODO - test with stub total calc
+    it 'calculates simple total' do
+      subject.order = ["1", "1"]
+      subject.calculate_total
+      expect(subject.total).to eq 0.5
+    end
+    it 'calculates simple total' do
+      subject.order = ["2", "1"]
+      subject.calculate_total
+      expect(subject.total).to eq 0.25
```

Comment:
You have tested all options together, which maximises confidence but may take extra time. Maybe two tests would fill you with the confidence you need 👩🏼‍🌾